lnwallet/btcwallet/btcwallet.go

```diff
@@ -710,6 +710,16 @@ func (b *BtcWallet) ListAccounts(name string,
 	return res, nil
 }
 
+// RequiredReserve returns the required reserve for a wallet.
+func (b *BtcWallet) RequiredReserve(
+	numAnchorChans int) btcutil.Amount {
+
+	anchorChanReservedValue := lnwallet.AnchorChanReservedValue
+	reserved := btcutil.Amount(numAnchorChans) * anchorChanReservedValue
```

Comment:
This value should be capped at `lnwallet.maxAnchorChanReservedValue` (needs to be exported as well).